sql/src/main/java/org/elasticsearch/action/bulk/BulkShardProcessor.java

```
@@ -225,6 +225,7 @@ private ShardId shardId(String indexName, String id, @Nullable String routing) {
                 routing
             ).shardId();
         } catch (IndexNotFoundException e) {
+            LOGGER.warn(String.format(Locale.ENGLISH, "Trying to insert value into unavailable partition: %s", indexName));
```

@mfussenegger do you recommend that it's better to throw the exception if this happens?